baseline/OSPS-BR.yaml

```diff
@@ -57,6 +57,10 @@ controls:
           - reference-id: CM-5
           - reference-id: CM-7
           - reference-id: SI-7
+      - reference-id: UKSSCOP
+        entries:
```

Comment:
This feels a little tenuous, especially 2.1.2. I can certainly see an argument though, so I'll just leave this comment for now and see if anyone else weighs in. If not, I think we proceed as-is.

Reply:
Claim 2.1.2 "Users of the build environment are required to authenticate on a regular basis."  Is Partial coverage at best.  A means of ensuring that actors & inputs are trusted to to periodically authenticate them.

Reply:
I think this may be a distinction between commercial and OSS software -- in OSS, we expect that attackers can do a lot more in the build system than you would expect for commercial software.  So this hardening is roughly because OSS doesn't enforce 2.1.6 "Users with access to the build environment are regularly reviewed to ensure they still have a legitimate need".